apiservers/engine/API-COMPATIBILITY.md

```diff
@@ -1,7 +1,73 @@
 # Docker Remote API Compatibility
 VIC attempts to be compatible with the Docker remote API; however, there are some differences.  The API differences are documented below, grouped together by functional areas.
 
-## Container APIs
+# support Matrix
+
+## Container API Support
+
+|API|support?|
+|---|---|
+|List Containers|YES|
+|Create a Container|YES|
+|inspect a container| YES|
+|List processes running inside a container|YES|
+|Get Container Logs|YES|
+|Inspect changes on a containers filesystem|YES|
+|Export a container|YES - In the future|
+|Get Container stats based on resource usage|__NO__|
+|Resize a container TTY|__NO__|
+|Start a Container|YES|
+|Stop a container|YES|
+|Restart a Container|YES|
+|kill a container|YES|
+|Update a Container|*Maybe*|
+|Rename a Container|YES|
+|Pause a Container|*maybe*|
+|Unpause a Container|*maybe*|
+|Attach a Container|YES|
+|Attach a Container(websocket)|YES|
+|wait a container|*maybe*|
```

Comment:
I think we may have to implement this after all, given the use of it by docker attach.